### PR TITLE
Email Verification Added

### LINKED ISSUE #38: Add Email Verification for new users using Firebase

### PR DESCRIPTION
**Pull request by SamarpanCoder2002**
# Description :

1. Email Verification Added.
2. After Sign-up user Navigate to log-in page and after email verification user can navigate to the home page.

## Fixes #38 :

Replace `#38` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/pratyushmp/Batua/blob/main/Contributing.md)?

- [x] Yes
- [ ] No

## Type of change :

_Please delete/edit options that are not relevant._

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist :

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

<p align="center">

</p>

If this pull request ok, please review and merge it..... Thanks @pratyushmp

### CODE REVIEW

**Review by pratyushmp — CHANGES_REQUESTED** [review]
- After the user signs up, instead of showing the user a dialogue box saying Proceed with login. It would be better if it informed that a verification email is sent to your registered email id. Verify and Proceed to login.
- Could you add the padding to the "Hello There ." text in the login screen so that it matches the UI in the readme.

**Comment by SamarpanCoder2002** [comment]
> * After the user signs up, instead of showing the user a dialogue box saying Proceed with login. It would be better if it informed that a verification email is sent to your registered email id. Verify and Proceed to login.
> * Could you add the padding to the "Hello There ." text in the login screen so that it matches the UI in the readme.

Paddding there means some spaces from uppersie of that text to look good right just like README.....????? I will make changes within an hour....thanks @pratyushmp

File: lib/sign_up.dart [hunk 1]
```diff
@@ -250,7 +250,7 @@ class SignUpScreenState extends State<SignUpScreen> {
                                     style: TextStyle(color: Colors.white),
                                   ),
                                   content: Text(
-                                    "Please Log-In To Continue",
+                                    "Verification Link Send to Your Registered Mail\nPlease Verify and Log-In",
```
Comment (pratyushmp) [inline lib/sign_up.dart:253]: change "Send" to "Sent".

**Review by pratyushmp — CHANGES_REQUESTED** [review]

File: lib/sign_up.dart (hunk 1 above)
Comment (SamarpanCoder2002) [reply lib/sign_up.dart]: Resolved

**Review by pratyushmp — APPROVED** [review]

**Comment by SamarpanCoder2002** [comment]
> * After the user signs up, instead of showing the user a dialogue box saying Proceed with login. It would be better if it informed that a verification email is sent to your registered email id. Verify and Proceed to login.
> * Could you add the padding to the "Hello There ." text in the login screen so that it matches the UI in the readme.

Already Made changes..... please check it @pratyushmp 




Hope you review it and merge it @pratyushmp @infiniteoverflow @nirala96

**Comment by pratyushmp** [comment]
Good Job @SamarpanCoder2002 .

**Comment by SamarpanCoder2002** [comment]
> Good Job @SamarpanCoder2002 .

Thank u @pratyushmp ... I requested in other issue about adding profile picture.....kindly check this

**Pull request merged**